### PR TITLE
Pick the right pup for salvaged memories.

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

File: pack/sm.json [hunk 1]
```diff
@@ -93,21 +93,21 @@
...
         "faction_cost": 3,
-        "flavor": "\"He only likes to eat live meat.\" -MaxX",
+        "flavor": "\"Its bytes are definitely worse than its bark.\" -Chaos Theory",
         "illustrator": "Liiga Smilshkalne",
-        "keywords": "Icebreaker - Killer",
+        "keywords": "Icebreaker - Fracter",
         "memory_cost": 1,
-        "pack_code": "sm",
-        "position": 6,
+        "pack_code": "atr",
```
Comment (gereons) [inline pack/sm.json]: should be "sm"

File: pack/sm.json [hunk 2]
```diff
@@ -93,21 +93,21 @@
...
+        "position": 99,
```
Comment (gereons) [inline pack/sm.json]: should be 6

**Review by gereons — CHANGES_REQUESTED** [review]
Oof! Thanks for catching this, I must've been blind when copy-pasting.

**Review by gereons — APPROVED** [review]

**Comment by plural** [comment]
thanks for the lightning fast review!  :D

**Pull request merged**